TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/RunningActivity.java

```diff
@@ -190,7 +198,7 @@ private void switchTangoLight(final TangoStatus status) {
         runOnUiThread(new Runnable() {
             @Override
             public void run() {
-                if (status == TangoStatus.SERVICE_RUNNING) {
+                if (status == TangoStatus.SERVICE_RUNNING || status == TangoStatus.SERVICE_CONNECTED) {
```

Comment:
Good point.
Just realized that this is related to #187, so it can be solved later when tackling this issue.
Sorry for the noise.